test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -91,16 +126,17 @@ public void testCreateDeleteKeyValueTable() {
         Assert.assertTrue(created);
         val segments = this.controller.getCurrentSegmentsForKeyValueTable(kvt.getScope(), kvt.getKeyValueTableName()).join();
         Assert.assertEquals(DEFAULT_CONFIG.getPartitionCount(), segments.getSegments().size());
+
         for (val s : segments.getSegments()) {
             // We know there's nothing in these segments. But if the segments hadn't been created, then this will throw
             // an exception.
-            this.tableStore.get(s.getScopedName(), Collections.singletonList(new ByteArraySegment(new byte[1])), TIMEOUT).join();
+            this.tableStore.get(s.getKVTScopedName(), Collections.singletonList(new ByteArraySegment(new byte[1])), TIMEOUT).join();
         }
 
         // Verify re-creation does not work.
         Assert.assertFalse(this.controller.createKeyValueTable(kvt.getScope(), kvt.getKeyValueTableName(), DEFAULT_CONFIG).join());
-
         // Delete and verify segments have been deleted too.
+        /*
```

Comment:
Uncomment this. If Delete is not implemented, add a TODO with a GitHub issue reference.